linkerd/app/inbound/src/lib.rs

```diff
@@ -227,28 +232,33 @@ pub fn spawn<P>(
             DispatchDeadline::after(dispatch_timeout)
         }))
         .push(errors::layer())
-        .push(trace::layer(
-            |src: &Source| info_span!("source", peer_ip = %src.remote.ip(), peer_id=?src.tls_peer),
-        ))
+        .push(trace::layer(|src: &tls::accept::Meta| {
+            info_span!(
+                "source",
+                peer.ip = %src.addrs.peer().ip(),
+                peer.id = ?src.peer_identity,
+                target.addr = %src.addrs.target_addr(),
```

Comment:
Nit/TIOLI: i notice we have just the peer's IP, but the target's `addr`, which includes both the IP and port; should these be formatted more consistently (e.g. `{ peer.addr, target.addr }` or `{ peer.ip, peer.port, target.ip, target.port }`)?

Reply:
We don't care about the peer's port because it's an ephemeral client port. Just as the inbound omits all of the peer information (because it's always localhost, and an ephemeral port.).  The logs need to be useful to humans, not machines.